app/(frontend)/auth/login/page.tsx

```diff
@@ -3,20 +3,56 @@
 import Image from "next/image"
 import {Button, Input} from "@nextui-org/react";
 import {Query} from "@/app/lib/api/Query";
+import {useRouter} from "next/navigation";
+import {useCallback, useState} from "react";
+import {ILoginFormError} from "@/app/lib/api/error/ApiError";
+import Link from "next/link";
+import {useCookies} from "react-cookie";
+import {AUTHORIZATION} from "@/app/lib/definitions";
+
+const DAYS = 1
+const MAX_AGE = 60 * 60 * 24 * DAYS
+
 
 export default function LoginPage() {
 
+    const router = useRouter()
+    const [state, setState] = useState<ILoginFormError | string>()
+    const [cookies, setCookie, removeCookie] = useCookies([AUTHORIZATION])
+
     const handleSubmit = async (formData: FormData) => {
         const query = new Query('/auth/login')
         const response = await query.withMethod('POST')
             .withBody(formData)
-            .send<{token: string}>()
-        console.log(response.data)
+            .send<{ token: string } & ILoginFormError>()
```

Comment:
I haven't noticed this at first, but now I'm not going to complain, I'll just leave a message for you:
When you're working with types, it's much easier and much more readable to create a new interface for this even if you won't reuse it. In this case you would need to create an ILoginResponse interface which extends the ILoginForm interface, so it would inherit it's properties. It's okay if you leave it like this, if you don't want to "refactor" it again, just keep in mind that it might be a good practice in the future.